tests/components/notify/test_file.py

```diff
@@ -25,8 +25,9 @@ def tearDown(self):  # pylint: disable=invalid-name
 
     def test_bad_config(self):
         """Test set up the platform with bad/missing config."""
+        # Platform should not be set up, but component should be set up.
         with assert_setup_component(0):
-            assert not setup_component(self.hass, notify.DOMAIN, {
+            assert setup_component(self.hass, notify.DOMAIN, {
```

Comment:
With the change to notify component this test doesn't add value anymore, I think. The notify component will be set up even with bad config, due to config validation removing the bad parts.